app/src/main/java/com/it3048/mobiledeviceproject/MeetingAdapter.kt

```diff
@@ -7,24 +7,24 @@ import android.view.ViewGroup
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 
-class MeetingAdapter(val c:Context, val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.meetingViewHolder>() {
-    inner class meetingViewHolder(val v:View):RecyclerView.ViewHolder(v){
+class MeetingAdapter(val c:Context, private val meetingList:ArrayList<MeetingDAO>):RecyclerView.Adapter<MeetingAdapter.MeetingViewHolder>() {
+    inner class MeetingViewHolder(private val v:View):RecyclerView.ViewHolder(v){
```

Comment:
changed access modifier and class name to follow coding conventions.
https://kotlinlang.org/docs/coding-conventions.html#class-headers